src/components/ModalEdit/index.js

```diff
@@ -0,0 +1,133 @@
+import { useContext } from "react";
+import DatePicker from "react-datepicker";
+import { Modal, Button } from "react-bootstrap";
+import DoctorsContext from "../../contexts/DoctorsContext";
+
+const ModalEdit = ({
+  setItemToEdit,
+  updateVisit,
+  editItem,
+  errors,
+  setErrors,
+}) => {
+  const setEditedVisitPartials = (visitFieldData) => {
+    setItemToEdit({ ...editItem, ...visitFieldData });
+  };
+
+  const doctors = useContext(DoctorsContext);
+  let { patient_name, text, date, doctor } = editItem;
+  const { name: selectedDoctorName, id: selectedDoctorId } = doctor;
+  date = new Date(date);
+  const [day, month, year] = [
+    date.getDate() < 10 ? `0${date.getDate()}` : date.getDate(),
+    date.getMonth() + 1 < 10 ? `0${date.getMonth() + 1}` : date.getMonth() + 1,
+    date.getFullYear(),
+  ];
+
+  const cancel = () => {
```

Comment:
use the correct names

Reply:
what's wrong with this name ?

Reply:
write down what this function is responsible for, what it cancels